app/views/posts/index.html.erb

```diff
@@ -1,9 +1,37 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>All Posts by User</title>
-</head>
-<body>
-  <h1>Here is a list of posts for a given user</h1>
-</body>
-</html>
\ No newline at end of file
+<h1>Here is a list of posts for a given user</h1>
+<%= link_to user_path(@user), class: "user-row-link" do %>
+  <div class="user-photo">
+    <%= image_tag @user.photo %>
+  </div>
+  <div class="user-row">
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <div class="post-counter">Number of posts: <%= @user.posts.count %></div>
+    </div>
+  </div>
+<% end %>
+
+<% @user.posts.each do |post| %>
+  <%= link_to user_post_path(@user, post), class: "post-link" do %>
+    <div class="post-row">
+      <h3><%= post.title %></h3>
+      <p><%= truncate(post.text, length: 50) %></p>
+      <div class="post-stats">
+        <span>Comments: <%= post.comments.count %></span>
+        <span>Likes: <%= post.likes.count %></span>
+      </div>
+    </div>
+  <% end %>
+
+  <div class="comments-container">
+    <h2>Recent Comments</h2>
+    <% post.comments.each do |comment| %>
+      <div class="comment">
+        <span class="comment-author"><%= comment.user.name %>:</span>
+        <span class="comment-text"><%= comment.text %></span>
+      </div>
+    <% end %>
+  </div>
+<% end %>
+
+<%= link_to 'Create a new post', new_user_post_path(@user), class: 'see-all-posts-btn' %>
```

Comment:
- [ ] As requested by the previous reviewer kindly use your `most_recent_comments` in your Post model to display the 5 most recent comments on the Post Show page.

Your view
![image](https://github.com/arnoldnekemiah/blog/assets/81932478/dddb77d8-caa8-421c-9aef-eeb6506b5371)

Expected view
![image](https://github.com/arnoldnekemiah/blog/assets/81932478/b3ecbb81-2100-4dc4-a017-5b48f7bbdbe4)

#### As you can see in the given wireframe only the 5 most recent comments are being displayed on the User post index page. Kindly add the recent comment method created in your Post model to this file so as to achieve the same

Reply:
Hey @Munsa1, code reviewers a mentioning the same issue which I corrected and lead to more changes required, I have run out of code review chances on this issue. From what I see my output is similar to the wireframe

Reply:
Hi @arnoldnekemiah  I understand your frustration but you have to use the recent comments method in your Post index so as to return the 5 most recent comments for each post. This is a requirement that has been pointed out by the previous reviewers.

If you need help slack me so I can assist @Munsa Mibenge